### ISSUE TITLE
Do this also support real-time (streaming) transcription with Whisper?

### ISSUE THREAD

**Issue by cerupcat** [codifilo/lorito#1]
Great job on the library. I was wondering if this also could work for streaming audio or not.

**Comment by codifilo** [comment]
No, unfortunately this library won't support streaming audio. I believe that's feature that Whisper should support itself and it'll be very hard to support from third party libraries. I tried the algorithm available in the stream.cpp file found in the whisper.cpp repository but the results are poor.
